api/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "api",
```

Comment:
Update name, description and other attributes of the this project.

possibly - jalantechnologies-marketing-website-boilerplate or something